packages/react/src/components/OverflowMenu/index.jsx

```diff
@@ -1,12 +1,15 @@
 import * as React from 'react';
 import { OverflowMenu as CarbonOverflowMenu } from 'carbon-components-react';
 import PropTypes from 'prop-types';
-import { getMenuOffset } from 'carbon-components-react/lib/components/OverflowMenu/OverflowMenu';
+// no idea why import { getMenuOffset } won't work, but it causes an 'getMenuOffset' is not exported by node_modules/carbon-components-react/lib/components/OverflowMenu/OverflowMenu.js error
+import * as OM from 'carbon-components-react/lib/components/OverflowMenu/OverflowMenu';
```

Comment:
This may have caused some issues downstream. Wild card imports are not suggested.